### PR TITLE
favfreak

### LINKED ISSUE #2888: [Tool Request] FavFreak
- **Name**: FavFreak
- **Source URL**: https://github.com/devanshbatham/FavFreak

### PR DESCRIPTION
**Pull request by dualfade**
close #2888

### CODE REVIEW

File: packages/favfreak/favfreak.install [hunk 1]
```diff
@@ -0,0 +1,13 @@
+post_install() {
+  set -e
+  cd /usr/share/favfreak
+  python -m virtualenv venv
```
Comment (noraj) [inline packages/favfreak/favfreak.install:4]: virtualenv is reserved for as the last resort packaging way for complex software with tons or very unmaintained dependencies.

favfreak has only one dependency: [mmh3](https://pypi.org/project/mmh3/) and this dependency is pure python no sub-dependencies, so very easy to package.

Package favfreak with [this template](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python) and mmh3 with [this template](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib).

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/favfreak/favfreak.install (hunk 1 above)
Comment (dualfade) [reply packages/favfreak/favfreak.install:4]: Ok Np will go to PKGBUILD only. I was using virtualenv install instructions from github.

File: packages/favfreak/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+pkgname=favfreak
+_pkgname=FavFreak
+pkgver=19.3a0bf2a
+pkgrel=1
+pkgdesc='Weaponizing favicon.ico for BugBounties , OSINT and what not.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon' 'blackarch-fingerprint')
+url='https://github.com/devanshbatham/FavFreak'
+license=('custom:unknown')
```
Comment (noraj) [inline packages/favfreak/PKGBUILD:12]: When no license contacting upstream is important https://github.com/devanshbatham/FavFreak/issues/6

File: packages/favfreak/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+depends=('python' 'python-virtualenv' 'python-mmh3')
+makedepends=('git' 'python-pip')
```
Comment (noraj) [inline packages/favfreak/PKGBUILD:14]:
```diff
-python-pip
+python-setuptools
```

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/favfreak/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/favfreak/PKGBUILD:12]: no update from from @devanshbatham as of yet. Will update once we have this.

Comment (dualfade) [reply packages/favfreak/PKGBUILD:12]: updated. Applying

**Pull request closed without merging**